tests/hmm/plotting.py

```diff
@@ -1092,8 +1092,8 @@ def PrintDatacard(categories, dict_procs, era, event_counts, filenames, ofname):
         dcof.write("RZ rateParam {0} dy_2j 1 \n".format(cat.full_name)) 
         #dcof.write("REWZ rateParam {0} ewk_lljj_mll50_mjj120 1 \n".format(cat.full_name))
     elif ("h_peak" in cat.full_name) or ("h_sideband" in cat.full_name):
-        dcof.write("R_01j_{1} rateParam {0} dy_m105_160_01j 1 \n".format(cat.full_name,str(era)))           
-        dcof.write("R_2j_{1} rateParam {0} dy_m105_160_2j 1 \n".format(cat.full_name,str(era)))
+        dcof.write("R_01j_{1} rateParam {0} dy_m105_160_01j 1 [0.1,5] \n".format(cat.full_name,str(era)))           
+        dcof.write("R_2j_{1} rateParam {0} dy_m105_160_2j 1 [0.1,5] \n".format(cat.full_name,str(era)))
```

Comment:
this range is here rather wide, shouldn't it be constrained more around 1? For eg 0.5 to 1.5 ?

Reply:
Hi Irene, good question. If in the toy mass scan study, I see cases where the rateParam for DY is fitted to even ~0.2 So I'm trying to keep it wide here. but yeah if you see issues with 0.1~5 later we can change it to a smaller range.

Reply:
ok sounds good